src/domain/state.ts

```diff
@@ -272,23 +273,43 @@ export class ApplicationStateManager {
       throw new Error('Operation not allowed');
     }
 
-    collaborator.id = new Date().getTime().toString();
-    collaborator.meta = {
+    const defaultCollaboratorInfo = {
```

Comment:
1- do this default merge before the validation of collaborator
2-add the type here please
`const defaultCollaboratorInfo: PersonalInfo`